src/main/java/core/basesyntax/WorkWithEnterList.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class WorkWithEnterList {
+    public WorkWithEnterList(OperationsStrategy operationsStrategy) {
+        this.operationsStrategy = operationsStrategy;
+    }
+
+    private OperationsStrategy operationsStrategy;
+
+
+    public String workWithEnterList(List<String> entery) {
+        StringBuilder str = new StringBuilder();
+

```

Comment:
There is a typo in the variable name `enetr`. It should be `entry` to improve readability and consistency.